codegenerator/cli/templates/static/codegen/src/LoadLayer.res

```diff
@@ -23,14 +23,16 @@ module BatchQueue = {
     byId: dict<loadSingle>,
     byIndex: dict<loadIndex>,
     entityMod: module(Entities.InternalEntity),
+    inMemTable: InMemoryTable.Entity.t<Entities.internalEntity>,
```

Comment:
I still don't think that inMemTable should be in state, IMO it should be passed as an argument all the way through schedule/execute.

To me this data structure suggests that the BatchQueue owns the inMemTable which it doesn't. It's difficult to express or enforce this in rescript without using immutable data structures (since the language doesn't have value semantics) which is why I think the semantics are important here.

Reply:
It's not possible to pass `inMemTable` to the schedule function. But I can do this with `inMemoryStore`. Should I?

Reply:
I actually think it's a subtle difference in reasoning about the code. For me, these two options are approximately the same

Reply:
> It's not possible to pass `inMemTable` to the schedule function. But I can do this with `inMemoryStore`. Should I?

Sorry I confused the two but yes I think this should be the case. A getter function for the table from the inMemoryStore can be in the BatchQueue state and then pass the inMemoryStore via argument. This would be my preference unless you think there's good reason otherwise.

We can actually have this getter function on the Entity module type. There's quite a few places this pattern gets used.